.gitignore

```diff
@@ -18,6 +18,7 @@
 /npm-debug.log*
 /testem.log
 /yarn-error.log
+development.env
```

Comment:
Do you need this? I didn't think Zola used one of these files

Reply:
I cannot recall why I had that. It's gone!